mkosi/resources/mkosi.md

```diff
@@ -2316,6 +2316,18 @@ these settings across reboots, copy
 `/etc/tmpfiles.d/static-nodes-permissions.conf` and change the mode of
 `/dev/kvm` from `0660` to `0666`.
 
+- How do I add a regular user to an image?
+
+You can use the following snippet in a post-installation script:
+
+```sh
+useradd --create-home --user-group $USER --password "$(openssl passwd -stdin -6 <$USER_PASSWORD_FILE)"
+```
+
+Note that from systemd v256 onwards, if enabled,
+`systemd-homed-firstboot.service` will prompt to create a regular user
+on first boot if there are no regular users.
```

Comment:
Can you indent the body by two spaces (same for the entry above) like this? This way the generated man page has the correct indentation as well.